tests/e2e/utils/utils.py

```diff
@@ -213,6 +213,10 @@ def load_yaml_file(file_path: str):
 
     return yaml.safe_load(content)
 
+def load_multiple_yaml_files(file_path: str):
```

Comment:
not used?

Reply:
It's used for the kustomize_to_helm_automation script. But I can remove it for this PR

Reply:
removed the function and will add it in the kustomize_to_helm_automation script PR